### PR TITLE
[user] create natural and legal

### PR DESCRIPTION
**Pull request by artmoni**
Create natural and legal Mangopay User

`MangoClient.create_user(%{...}, :legal)`
`MangoClient.create_user( %{...},:natural)`

### CODE REVIEW

File: lib/mangoex/client.ex [hunk 1]
```diff
@@ -18,10 +18,18 @@ defmodule Mangoex.Client do
     GenServer.call(@client_name, {:create_user, body})
   end
 
+  def create_user(type, body) do
+    GenServer.call(@client_name, {:create_user, body, type})
```
Comment (overture8) [inline lib/mangoex/client.ex]: I think `type` and `body` need swapped around here (based on the order of the related `handle_call`.

File: lib/mangoex/client.ex [hunk 2]
```diff
@@ -18,10 +18,18 @@ defmodule Mangoex.Client do
...
+  end
+
   def update_user(user_id, body) do
     GenServer.call(@client_name, {:update_user, user_id, body})
   end
 
+  def update_user(type, user_id, body) do
+    GenServer.call(@client_name, {:update_user, user_id, body, type})
```
Comment (overture8) [inline lib/mangoex/client.ex]: Again, I think `user_id, body, type` are not in the order the related `handle_calls` expect.

**Review by overture8 — CHANGES_REQUESTED** [review]

File: lib/mangoex/client.ex (hunk 1 above)
Comment (artmoni) [reply lib/mangoex/client.ex]: done :)

**Comment by artmoni** [comment]
@overture8 the last commit should be nice for legal user creation

**Pull request merged**